flang/test/Lower/OpenMP/omp-wsloop-collapse.f90

```diff
@@ -13,13 +13,13 @@ program wsloop_collapse
   integer :: i, j, k
   integer :: a, b, c
   integer :: x
-! FIRDialect:         %[[VAL_0:.*]] = fir.alloca i32 {bindc_name = "a", uniq_name = "_QEa"}
-! FIRDialect:         %[[VAL_1:.*]] = fir.alloca i32 {bindc_name = "b", uniq_name = "_QEb"}
-! FIRDialect:         %[[VAL_2:.*]] = fir.alloca i32 {bindc_name = "c", uniq_name = "_QEc"}
-! FIRDialect:         %[[VAL_3:.*]] = fir.alloca i32 {bindc_name = "i", uniq_name = "_QEi"}
-! FIRDialect:         %[[VAL_4:.*]] = fir.alloca i32 {bindc_name = "j", uniq_name = "_QEj"}
-! FIRDialect:         %[[VAL_5:.*]] = fir.alloca i32 {bindc_name = "k", uniq_name = "_QEk"}
-! FIRDialect:         %[[VAL_6:.*]] = fir.alloca i32 {bindc_name = "x", uniq_name = "_QEx"}
+! FIRDialect:         %[[VAL_0:.*]] = fir.alloca i32 {bindc_name = "a", uniq_name = "_QF_qqmainEa"}
```

Comment:
Lowercase "__qqmain" looks odd. How is this to be demangled? The demangling code appears to be missing.

`_QFEa` should be unique and not drag in a compiler generated unique symbol that gets erroneously converted to lowercase.

Reply:
It was demangled as a host named `_qqmain`, but I agree it's odd. Keeping the string empty for anonymous main program is a much better approach, thanks. I updated the PR and added some unit test with the empty host names.